sqlstore/src/main/resources/db/migration/V0028__model_refactor.sql

```diff
@@ -2,3 +2,28 @@ ALTER TABLE SampleAdditionalInfo ADD COLUMN groupId int(10);
 ALTER TABLE SampleAdditionalInfo ADD COLUMN groupDescription varchar(255);
 
 UPDATE SampleAdditionalInfo SET groupId = (SELECT groupId FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId), groupDescription = (SELECT groupDescription FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId);
+
+ALTER TABLE SampleAdditionalInfo DROP FOREIGN KEY `FKeutn2473w3yr16khgalspuviw`; -- createdBy fkey
+ALTER TABLE SampleAdditionalInfo DROP FOREIGN KEY `FKp8bvx3e7jsmnyw51toi7mq7cq`; -- updatedBy fkey
```

Comment:
It's the not FK names that are the problem. It's the comment (`-- createdBy fkey`) location that's the problem. When they're inline the migration doesn't work. Putting them on their own line solves the problem.